src/promptflow-evals/CHANGELOG.md

```diff
@@ -1,18 +1,20 @@
-# promptflow-evals package

```

Comment:
Did we want to delete this first line?

Reply:
Yes, I am trying to align our CHANGELOG.md with PF's.
https://github.com/microsoft/promptflow/blob/main/src/promptflow/CHANGELOG.md